src/mlpack/tests/main_tests/random_forest_test.cpp

```diff
@@ -25,7 +25,7 @@ using namespace mlpack;
 
 struct RandomForestTestFixture
 {
- public:
+public:
```

Comment:
I don't understand the changes you've made to this file---they are all style changes, and now the code does not match the style guidelines for the codebase.  Do you think you could revert changes to this file?

Reply:
Actually I use prettier in my text editor so it may have been a result of that. I will revert back the changes.

Reply:
Just FYI, these haven't yet been reverted. :+1:

Reply:
(This is still true now.)